src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1489,6 +1436,73 @@ public static function debug_data() {
 		return $info;
 	}
 
+	/**
+	 * Get the file system section of the debug data.
```

Comment:
```suggestion
	 * Gets the file system section of the debug data.
```
Minor change: Core uses third-person singular verbs for docblock summaries("Gets" not "Get", "Sets" not "Set", etc.)

[Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#documenting-tips)

Reply:
My bad. I was so sure that it was the other way round. I frequently mix that up.